tests/test_shortcuts.py

```diff
@@ -15,16 +15,16 @@
 
 class TestShortcuts(unittest.TestCase):
     def test_force_list(self):
-        self.assertEqual(force_list("a"), ["a"])
-        self.assertEqual(force_list(["a"]), ["a"])
-        self.assertEqual(force_list(["a", "b"]), ["a", "b"])
-        self.assertEqual(force_list(set(["a", "b"])), ["a", "b"])
+        self.assertItemsEqual(force_list("a"), ["a"])
+        self.assertItemsEqual(force_list(["a"]), ["a"])
+        self.assertItemsEqual(force_list(["a", "b"]), ["a", "b"])
```

Comment:
Can the first three cases continue using `assertEqual`? There might very well be programs assuming the that the a list will not change order.

Reply:
Sure, fixed.